packages/web/src/scss/components/Grid/_Grid.scss

```diff
@@ -22,6 +22,29 @@
             --grid-spacing-y: initial;
         }
     }
+
+    align-items: stretch;
```

Comment:
please remove these lines. This selector is here only to reset the CSS variables so they are not inherited in the nested Grid.